### ISSUE TITLE
Conflict when more than one instance on one page

### ISSUE THREAD

**Issue by zhengyi-yang** [iVis-at-Bilkent/cytoscape.js-auto-pan-on-out-of-canvas-bounds#8]
I have declared several Cytoscape instances on one page with different div as containers. And I called `cy.autopanOnDrag({});` for each Cytoscape instance. However, no matter which graph(the node in that graph) I drag, the viewport of the last Cytoscape instance moves. 

Here is a part of my code:

```javascript
 cys = $.map($('.cy'), function (container) {
        var cy = cytoscape($.extend({}, cytoscapeOptions, {container: container}));
        cy.autopanOnDrag({
            // options here...
        });
        return cy;
    });
```

**Comment by metincansiper** [comment]
I created a PR (https://github.com/iVis-at-Bilkent/cytoscape.js-autopan-on-drag/pull/7) to solve this problem.

**Comment by zhengyi-yang** [comment]
Thanks a lot @metincansiper

**Comment by kinimesi** [comment]
Fixed in PR #7
